lib/fluent/config/element.rb

```diff
@@ -71,6 +71,10 @@ def add_element(name, arg = '')
         e
       end
 
+      def merge_elements(elements)
+        @elements.concat(elements)
+      end
+
```

Comment:
Maybe we don't need this new method.

We can merge `Fluent::Config::Element` by

```rb
          @conf += Fluent::Config.build(config_path: path,
                                        encoding: @conf_encoding,
                                        use_v1_config: @use_v1_config,
                                        type: @config_file_type)
```